src/js/disability-benefits/526EZ/components/FormStartControls.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import LoadingIndicator from '@department-of-veterans-affairs/jean-pants/LoadingIndicator';
+import SaveInProgressIntro from '../../../common/schemaform/save-in-progress/SaveInProgressIntro';
+
+import { ITFErrorAlert, VerifiedAlert, UnauthenticatedAlert, UnverifiedAlert, ITFDescription } from '../helpers';
+
+export default function FormStartControls(props) {
+  const { user } = props;
+
+  const somethingWentWrong = props.ITFStatus && props.ITFStatus === 'expired' || // This may not be possible
+                             props.errors && props.errors.length;
+  return (
+    <div>
+      {!user.login.currentlyLoggedIn && <div>
+        {UnauthenticatedAlert}
+        <button className="usa-button-primary" onClick={props.authenticate}>Sign In and Verify Your Identity</button>
+      </div>}
+      {!user.profile.verified && user.login.currentlyLoggedIn && <div>
+        {UnverifiedAlert}
+        <a href={`/verify?next=${window.location.pathname}`} className="usa-button-primary verify-link">Verify Your Identity</a>
+      </div>}
+      {somethingWentWrong && ITFErrorAlert}
+      {props.ITFStatus === 'pending' && <LoadingIndicator message="Please wait while we check that your intent to file has been submitted."/>}
```

Comment:
Maybe this should read `...check if your intent...`? When this is shown, they're on the intro page, right? On the intro page, we won't have attempted to file an ITF yet, so it may make more sense to say that we're checking _if_ it's been filed rather than checking _that_ it's been filed.